package.json

```diff
@@ -44,14 +42,22 @@
     "@babel/cli": "^7.4.4",
     "@babel/preset-env": "^7.4.5",
     "@babel/register": "^7.4.4",
+    "babel-cli": "^6.26.0",
     "babel-plugin-istanbul": "^5.1.4",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "chai": "^4.1.2",
+    "chai-http": "^4.0.0",
     "cross-env": "^5.2.0",
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.11.0",
     "eslint-plugin-jasmine": "^2.10.1",
     "istanbul": "^0.4.5",
     "jasmine": "^3.4.0",
```

Comment:
Also, remove all unused modules in your dependencies section. also, separate dependencies from dev dependencies